source/dubregistry/repositories/bitbucket.d

```diff
@@ -83,7 +87,7 @@ class BitbucketRepository : Repository {
 	RepositoryFile[] listFiles(string commit_sha, InetPath path)
 	{
 		assert(path.absolute, "Passed relative path to listFiles.");
-		auto url = "https://bitbucket.org/api/2.0/repositories/"~m_owner~"/"~m_project~"/src/"~commit_sha~path.toString()~"?pagelen=100";
+		auto url = getAPIURLPrefix ~ "/api/2.0/repositories/"~m_owner~"/"~m_project~"/src/"~commit_sha~path.toString()~"?pagelen=100";
```

Comment:
I just noticed the prefixes changed, in the old code sometimes api.bitbucket.org and sometimes bitbucket.org/api was used. Is this documented somewhere which one to use now?